docs/labwc-actions.5.scd

```diff
@@ -180,6 +180,46 @@ Actions are used in menus and keyboard/mouse bindings.
 	*wrap* [yes|no] Wrap around from last desktop to first, and vice
 	versa. Default yes.
 
+*<action name="VirtualOutputAdd" output_name="value" />*
+	Add virtual output (headless backend).
+
+	Virtual outputs is a useful mechanism. For example, it can be used to overlay
+	virtual output on real output, but with a different resolution (this can be
```

Comment:
Suggest removing the first sentence about the usefulness.

Could be rewritten like
> Virtual outputs can be used for example to overlay a virtual output on a real output, but with a different resolution [...]